### ISSUE TITLE
Bear glTF 2.0 model is not displayed in Cesium.js v1.40 or later

### ISSUE THREAD

**Issue by cx20** [CesiumGS/cesium#6416]
I tried to display the bear glTF model that was in sketchfab with the latest version of Cesium.js.
https://sketchfab.com/models/f2f13a8630004b1c82730d8b9ffa0e1f

However, in Cesium.js v1.40 and later, an error is displayed and it seems that it is not displayed correctly.

[Cesium.js v1.39 + Bear glTF model result:](http://jsdo.it/cx20/0r7J)
![image](https://user-images.githubusercontent.com/1833979/38254816-2431edd8-3795-11e8-8198-6b476c4ede0a.png)

[Cesium.js v1.44 + Bear glTF model result:](http://jsdo.it/cx20/wuVp)
```
Cesium.js:532 
An error occurred while rendering.  Rendering has stopped.
undefined
TypeError: i.clampTime is not a function
TypeError: i.clampTime is not a function
    at Array.<anonymous> (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:497:22244)
    at h (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:496:19348)
    at c.update (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:496:21384)
    at Ae.update (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:498:20408)
    at a.update (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:492:8011)
    at tt (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:525:21935)
    at Ke (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:525:19739)
    at Qe (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:525:17745)
    at lt (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:525:25765)
    at ut (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:525:25933)
```

**Comment by lilleyse** [comment]
I opened a fix in #6422

**Comment by cx20** [comment]
Although it may be another problem, in Cesium.js v1.42 or later, it seems that animation-enabled models are not animated. BoxAnimated, CesiumMan, Monster, etc..

**Comment by emackey** [comment]
@cx20 There was a change there, add `shouldAnimate: true` to the `Cesium.Viewer` constructor options.  The default behavior starts in pause now.

**Comment by cx20** [comment]
I noticed that the following glTF model can not be displayed. I will report this as a separate Issue.

https://cdn.rawgit.com/mrdoob/rome-gltf/784089b4/files/models/life_soup/quadruped_fox.gltf
https://cdn.rawgit.com/pissang/claygl/c4f45119/example/assets/models/SambaDancing/SambaDancing.gltf

**Comment by pjcozzi** [comment]
@cx20 @emackey thanks for reporting all these glTF issues!  OK to close this one?

**Comment by emackey** [comment]
@pjcozzi The related PR #6422 didn't get merged yet.  @lilleyse is it being held up by the lack of a test model?

**Comment by emackey** [comment]
This auto-closed because I merged the animation fix.  I think I'm still seeing an issue where the bear renders inside-out when viewed from the back.  Not sure what that's about...

**Comment by emackey** [comment]
The side view just shows some slicing through the balloons.  The rear view is so bad it looks like you're seeing the front of the bear.

![brokenbearonballoons](https://user-images.githubusercontent.com/1235080/39402359-0b7a66d6-4b2b-11e8-928c-189010ef563e.png)

**Comment by mramato** [comment]
@emackey do you think this is fixed by #6513

**Comment by emackey** [comment]
> do you think this is fixed by #6513

Sadly no.  I just merged master into that branch, but the artifacts look the same.

**Comment by emackey** [comment]
I've run through a bunch of other models, and haven't found any other than the bear that display this particular problem.  Also, the problem only appears when animations are enabled.  So... some bad interaction between something animated and the diffs in 417c3f9, it would seem.

**Comment by lilleyse** [comment]
It might be https://github.com/AnalyticalGraphicsInc/cesium/issues/6447.

We never figured out why log depth and skinned meshes don't play well together. The current workaround is an improvement but sometimes reveals artifacts like this.

**Comment by bagnell** [comment]
Yes, that's the issue. If you remove the workaround, this model works fine. I'm looking into it.

**Comment by mramato** [comment]
@emackey  Did we check to see if this model has the same normalized bone weights problem as Cesium Man?

It looks unlikely that we will get to this for the 1.45 release today and just want to make sure the issue is model specific or at least not the sign of a larger problem.

**Comment by emackey** [comment]
No, the Bear model has the opposite problem, it is being damaged by a workaround that's in place for the sake of the broken Cesium Man model.

Let's not hold the release for this.
